x-pack/plugins/security_solution/public/management/pages/policy/view/integration_tests/policy_list.test.tsx

```diff
@@ -231,19 +233,19 @@ describe.skip('When on the policy list page', () => {
       });
 
       // change pageSize
-      await act(async () => {
-        (await renderResult.getByTestId('tablePaginationPopoverButton')).click();
+      act(() => {
+        renderResult.getByTestId('tablePaginationPopoverButton').click();
```

Comment:
Consider using userEvent to invoke user actions like click. You won't have to wrap the event inside an `act` that way. Something like:
```
userEvent.click(renderResult.getByTestId('someId'))
```

Reply:
Done, thanks for the idea!

Also, it turned out that this (1) works:
```typescript
      userEvent.click(renderResult.getByTestId('tablePaginationPopoverButton'));
      const pageSize10 = await renderResult.findByTestId('tablePagination-10-rows'); // <<<----
      userEvent.click(pageSize10);
```

but this (2) throws an error:
```typescript
      userEvent.click(renderResult.getByTestId('tablePaginationPopoverButton'));
      const pageSize10 = renderResult.getByTestId('tablePagination-10-rows'); // <<<----
      userEvent.click(pageSize10);
```

because the button for `pageSize10` lives in a pop-up, and after opening it, a small time needs to pass until the `pointerEvents` computed property is recalculated, and changes from `none` to `auto`. So the `async` solution (1) theoretically runs _after_ the computed property is recalculated, but to avoid this kind of flakiness, I used this solution that was used elsewhere in our codebase:
```typescript
      userEvent.click(renderResult.getByTestId('tablePaginationPopoverButton'));
      const pageSize10 = renderResult.getByTestId('tablePagination-10-rows');
      userEvent.click(pageSize10, undefined, { skipPointerEventsCheck: true });
```
It skips the check for the `pointerEvents` property.

Reply:
[3e6c562](https://github.com/elastic/kibana/pull/150204/commits/3e6c5628516df27947f05f39c9b2a0a7a879a0bd)

Reply:
You can instead use `waitForEuiPopoverOpen` before you get the `pageSize10` node. We do something similar [here](https://github.com/ashokaditya/kibana/blob/4076ecef2d4a6f88ae49731f00d8fcbf0adbcc11/x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/response_actions_log.test.tsx#L379)

Reply:
wow, that's a very useful one, thanks! here's the fix: [3c36fd2](https://github.com/elastic/kibana/pull/150204/commits/3c36fd28752faf02aa8db3b4c39b2ec342eea4bb)

Reply:
Thanks for the changes!